### PR TITLE
Math Magicians Components

### PR DESCRIPTION
**Pull request by ab-noori**
## Math Magicians: Components

  ###  **🌟**Features:
- Linters files and configurations are added.
- Boilerplate data and files are removed.
- Components directory is created.
- [`Calculator`](https://github.com/ab-noori/math_magicians/blob/5396acefd1597bf8db4eb2b6a57556a3f543f148/src/components/Calculator.jsx) component is added.
- Calculator component is styled [`App.scss`](https://github.com/ab-noori/math_magicians/blob/5396acefd1597bf8db4eb2b6a57556a3f543f148/src/App.scss).
### [ **🚀** Project Demo](https://ab-noori.github.io/math_magicians/)

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    </div>
+    <div className="row">
+      <button type="button" className="btn zero number">0</button>
+      <button type="button" className="btn decimal">.</button>
+      <button type="button" className="btn equals operator">=</button>
+    </div>
+  </div>
+);
+
+export default Calculator;
```
Comment (julie-ify) [inline src/components/Calculator.jsx]: - [ ] The Calculator component should have at least one child component. Kindly add that.

File: src/logo.svg [hunk 2]
```diff
@@ -1 +0,0 @@
...21.2 430c-9.3-9.6-18.6-20.3-27.8-32 9 .4 18.2.7 27.5.7 9.4 0 18.7-.2 27.8-.7-9 11.7-18.3 22.4-27.5 32zm-74.4-58.9c-14.2-2.1-27.9-4.7-41-7.9 3.7-12.9 8.3-26.2 13.5-39.5 4.1 8 8.4 16 13.1 24 4.7 8 9.5 15.8 14.4 23.4zM420.7 163c9.3 9.6 18.6 20.3 27.8 32-9-.4-18.2-.7-27.5-.7-9.4 0-18.7.2-27.8.7 9-11.7 18.3-22.4 27.5-32zm-74 58.9c-4.9 7.7-9.8 15.6-14.4 23.7-4.6 8-8.9 16-13 24-5.4-13.4-10-26.8-13.8-39.8 13.1-3.1 26.9-5.8 41.2-7.9zm-90.5 125.2c-35.4-15.1-58.3-34.9-58.3-50.6 0-15.7 22.9-35.6 58.3-50.6 8.6-3.7 18-7 27.7-10.1 5.7 19.6 13.2 40 22.5 60.9-9.2 20.8-16.6 41.1-22.2 60.6-9.9-3.1-19.3-6.5-28-10.2zM310 490c-13.6-7.8-19.5-37.5-14.9-75.7 1.1-9.4 2.9-19.3 5.1-29.4 19.6 4.8 41 8.5 63.5 10.9 13.5 18.5 27.5 35.3 41.6 50-32.6 30.3-63.2 46.9-84 46.9-4.5-.1-8.3-1-11.3-2.7zm237.2-76.2c4.7 38.2-1.1 67.9-14.6 75.8-3 1.8-6.9 2.6-11.5 2.6-20.7 0-51.4-16.5-84-46.6 14-14.7 28-31.4 41.3-49.9 22.6-2.4 44-6.1 63.6-11 2.3 10.1 4.1 19.8 5.2 29.1zm38.5-66.7c-8.6 3.7-18 7-27.7 10.1-5.7-19.6-13.2-40-22.5-60.9 9.2-20.8 16.6-41.1 22.2-60.6 9.9 3.1 19.3 6.5 28.1 10.2 35.4 15.1 58.3 34.9 58.3 50.6-.1 15.7-23 35.6-58.4 50.6zM320.8 78.4z"/><circle cx="420.9" cy="296.5" r="45.7"/><path d="M520.5 78.1z"/></g></svg>
```
Comment (julie-ify) [inline src/logo.svg:1]: - [ ] Good job deleting this file, but you still need to remove the listed boilerplate files in the public folder: `logo192.png`, `logo512.png`, `robots.txt`.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ab-noori,

Good job so far!

## Highlights!
- Good code organization and design ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logo.svg (hunk 2 above)
Comment (ab-noori) [reply src/logo.svg:1]: Hi @julie-ify **👋**,
Thank you so much **🙏**  for reviewing my project and giving constructive feedback.
I have added the [`Buttons`](https://github.com/ab-noori/math_magicians/tree/1e1c0c3d724bfcfc4d27250d433ba763f61bcc0b/src/components) component and removed unnecessary data from the code base.

**Review by OyePriscilla — APPROVED** [review]
Hi @ab-noori 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Calculator component has child component ✔️ 
Boiler template removed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ab-noori** [comment]
Hi @OyePriscilla  👋,
Thank you so much 🙏 for reviewing and approving my project.

**Pull request merged**